src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar start-day--wed month-length--28">
+      <div class="calendar__day--1"></div>
+      <div class="calendar__day--2"></div>
+      <div class="calendar__day--3"></div>
+      <div class="calendar__day--4"></div>
+      <div class="calendar__day--5"></div>
+      <div class="calendar__day--6"></div>
+      <div class="calendar__day--7"></div>
+      <div class="calendar__day--8"></div>
```

Comment:
Do not hardcode day number. use [nth-child](https://css-tricks.com/how-nth-child-works/)
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```